WcaOnRails/config/locales/en.yml

```diff
@@ -459,6 +459,14 @@ en:
         candidate_delegate: "Junior Delegate"
         delegate: "Delegate"
         senior_delegate: "Senior Delegate"
+      role_status:
```

Comment:
Can we further group this by the type of role? Something like...

```yaml
role_status:
  delegate:
    trainee_delegate: "Trainee Delegate"
    candidate_delegate: "Junior Delegate"
    ...
  teams_committees:
    member: "Team Member"
    senior_member: "Senior Member"
    leader: "Leader"
  probation:
    ...
```

Reply:
(note that the keys like `delegate`, `teams_committees`, etc. are provisional only for the sake of demonstration. The actual keys should match the group_type constants in the DB)

Reply:
Okay done. For teams_committees, for now I added them under teams. They don't have an entry in group_type yet. The group_type in mind are `teams`, `committees`, `councils`.